subsys/net/openthread/Kconfig.defconfig

```diff
@@ -169,11 +169,14 @@ config NET_PKT_TIMESTAMP
 
 # CSL Transmitter configuration
 config OPENTHREAD_PLATFORM_CSL_UNCERT
-	default 30 if BOARD_NRF54L15DK_NRF54L15_CPUAPP
+	default 30 if BOARD_NRF54L15DK_NRF54L15_CPUAPP || BOARD_NRF54L15DK_NRF54L15_CPUAPP_NS \
```

Comment:
Maybe it would be better to replace all listed boards with `SOC_SERIES_NRF54LX`?

Reply:
CSL accuracy and uncertainty is directly correlated with clock accuracy on the DK, so we cannot use SOC related configs. unfortunately I didn't find any Kconfig that is common for all nrf54l15DK based targets

Reply:
@adigie checked that we can use BOARD_NRF54L15DK . Thanks!